linkerd/app/core/src/profiles.rs

```diff
@@ -79,11 +86,16 @@ where
     type Stream = Rx;
 
     fn get_routes(&self, dst: &NameAddr) -> Option<Self::Stream> {
-        let (tx, rx) = mpsc::channel(1);
+        if !self.suffixes.iter().any(|s| s.contains(dst.name())) {
```

Comment:
I know this is just a move from elsewhere, but I'm confused by this logic.  We check that the suffix contains the name?  Isn't that backwards?

Reply:
we can change the word "contains", but a domain suffix contains its subdomains. they are constituents.